readme.md

```diff
@@ -1,7 +1,7 @@
 # Product cards
```

Comment:
Passes tests in terminal (npm test) - https://marekder.github.io/layout_product-cards/report/html_report/

Doesn't pass test on GitHub?